### PR TITLE
chore: add OpenTelemetry eBPF Profiler to the emerging apps list

### PR DESCRIPTION
**Pull request by florianl**
Elastic [donated](https://opentelemetry.io/blog/2024/elastic-contributes-continuous-profiling-agent) its eBPF based continious profiler, originated from optimyze, to Open Telemetry.
With this donation it got open sourced and is now available for everyone.

### CODE REVIEW

File: src/data/pages/applications/emerging.js [hunk 1]
```diff
@@ -440,6 +440,20 @@ const emergingApplications = [
...
+  {
+    logoUrl: 'https://github.com/open-telemetry/opentelemetry-ebpf-profiler',
+    name: 'OpenTelemetry eBPF Profiler',
+    logoName: 'otelLogo',
+    title: 'Whole-system, cross-language continuous profiler for Linux',
+    description: `The Open Telemetry eBPF-based continuous profiler offers comprehensive, low-overhead whole-system profiling for
+    Linux systems. It supports a wide range of programming languages, including native code without debug symbols, and provides
+    deep insights into application behavior. By leveraging the experimental OTel profiling signal and innovative techniques, it
+    delivers unparalleled visibility into complex software systems without intrusive instrumentation. This project empowers developers
+    to identify performance bottlenecks and optimize their applications efficiently.`,
```
Comment (xmulligan) [inline src/data/pages/applications/emerging.js:452]:
```suggestion
    deep insights into application behavior. By leveraging the experimental OTel profiling signals, this project empowers developers
    to identify performance bottlenecks and optimize their applications efficiently.`,
```

**Comment by xmulligan** [comment]
Thanks for the addition, one comment and looks like the build currently is failing

**Review by xmulligan — APPROVED** [review]

**Comment by xmulligan** [comment]
I'm fine to merge once you can fix the build errors

**Comment by florianl** [comment]
I'm running into multiple issues preventing me from testing/building this locally.

```
$ npm run start

> ebpf-gatsby@0.1.0 start
> npm run develop


> ebpf-gatsby@0.1.0 develop
> gatsby develop -H 0.0.0.0

node:internal/modules/cjs/loader:453
      throw err;
      ^

Error: Cannot find module '/home/flehner/src/github.com/ebpf-io/ebpf.io-website/node_modules/utila/lib/utila.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:445:19)
    at Function.Module._findPath (node:internal/modules/cjs/loader:716:18)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1131:27)
    at Function.Module._load (node:internal/modules/cjs/loader:986:27)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/home/flehner/src/github.com/ebpf-io/ebpf.io-website/node_modules/dom-converter/lib/objectToSaneObject.js:5:10)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/home/flehner/src/github.com/ebpf-io/ebpf.io-website/node_modules/utila/package.json',
  requestPath: 'utila'
}

Node.js v20.16.0
```

And similar issues for `npm run build`, `npm run serve` and `npm run clean`, despite following [getting started](https://github.com/ebpf-io/ebpf.io-website?tab=readme-ov-file#getting-started). So I'm hoping, that I did identify the issue that caused the CI to fail and got it fixed.

File: src/data/pages/applications/emerging.js [hunk 2]
```diff
@@ -440,6 +440,19 @@ const emergingApplications = [
...
+    logoUrl: 'https://github.com/open-telemetry/opentelemetry-ebpf-profiler',
+    name: 'OpenTelemetry eBPF Profiler',
+    logoName: 'otelLogo',
+    title: 'Whole-system, cross-language continuous profiler for Linux',
+    description: `The Open Telemetry eBPF-based continuous profiler offers comprehensive, low-overhead whole-system profiling for
+    Linux systems. It supports a wide range of programming languages, including native code without debug symbols, and provides
+    deep insights into application behavior. By leveraging the experimental OTel profiling signals, this project empowers developers
+    to identify performance bottlenecks and optimize their applications efficiently.`,
+    urls: [
+      { label: 'GitHub', url: 'https://github.com/open-telemetry/opentelemetry-ebpf-profile' },
```
Comment (xmulligan) [inline src/data/pages/applications/emerging.js:453]:
```suggestion
      { label: 'GitHub', url: 'https://github.com/open-telemetry/opentelemetry-ebpf-profiler' },
```

**Pull request merged**